github/ipfs.yml

```
@@ -4875,7 +4876,6 @@ teams:
         - laurentsenta
         - marten-seemann
         - petar
```

I think `reidlw` didn't accept the previous invite.
```suggestion
        - petar
        - reidlw
```